diff -aur download/webkit/Source/JavaScriptCore/CMakeLists.txt target/webkit/Source/JavaScriptCore/CMakeLists.txt
--- download/webkit/Source/JavaScriptCore/CMakeLists.txt	2020-07-22 04:40:59.000000000 +0300
+++ target/webkit/Source/JavaScriptCore/CMakeLists.txt	2020-11-08 03:26:26.000000000 +0300
@@ -1368,7 +1368,7 @@
 )
 target_include_directories(LLIntOffsetsExtractor SYSTEM PRIVATE ${JavaScriptCore_SYSTEM_INCLUDE_DIRECTORIES})
 
-add_subdirectory(shell)
+# add_subdirectory(shell)
 
 target_include_directories(LowLevelInterpreterLib
     PRIVATE "$<TARGET_PROPERTY:JavaScriptCore,INCLUDE_DIRECTORIES>")
diff -aur download/webkit/Source/cmake/OptionsJSCOnly.cmake target/webkit/Source/cmake/OptionsJSCOnly.cmake
--- download/webkit/Source/cmake/OptionsJSCOnly.cmake	2020-06-12 23:52:02.000000000 +0300
+++ target/webkit/Source/cmake/OptionsJSCOnly.cmake	2020-11-08 03:24:43.000000000 +0300
@@ -37,11 +37,7 @@
 set(ENABLE_WEBKIT OFF)
 set(ENABLE_WEBINSPECTORUI OFF)
 
-if (WIN32)
-    set(ENABLE_API_TESTS OFF)
-else ()
-    set(ENABLE_API_TESTS ON)
-endif ()
+set(ENABLE_API_TESTS OFF)
 
 if (WTF_CPU_ARM OR WTF_CPU_MIPS)
     SET_AND_EXPOSE_TO_BUILD(USE_CAPSTONE TRUE)
