diff -aurN download/webkit/Source/WTF/wtf/PlatformUse.h target/webkit/Source/WTF/wtf/PlatformUse.h
--- download/webkit/Source/WTF/wtf/PlatformUse.h	2020-08-11 07:23:51.000000000 +0300
+++ target/webkit/Source/WTF/wtf/PlatformUse.h	2020-11-08 04:18:39.000000000 +0300
@@ -289,7 +289,7 @@
 #define USE_PLATFORM_SYSTEM_FALLBACK_LIST 1
 #endif
 
-#if PLATFORM(IOS_FAMILY) || (!(defined(USE_SYSTEM_MALLOC) && USE_SYSTEM_MALLOC) && (OS(LINUX) && (PLATFORM(GTK) || PLATFORM(WPE))))
+#if PLATFORM(IOS_FAMILY) || defined(__ANDROID__) || (!(defined(USE_SYSTEM_MALLOC) && USE_SYSTEM_MALLOC) && (OS(LINUX) && (PLATFORM(GTK) || PLATFORM(WPE))))
 #define USE_BMALLOC_MEMORY_FOOTPRINT_API 1
 #endif
 
